token.go

```diff
@@ -40,6 +52,7 @@ type Token struct {
 
 	BankAccount *BankAccount `json:"bank_account"`
 	Card        *Card        `json:"card"`
+	Account     *Account     `json:"account"`
```

Comment:
This is not something that exists in the API. While it's a Token you create about an account, it doesn't have an `account` property right now. Even if it did, it'd be a hash and not an `Account` because this is to collect PII client-side and then you add this to an account later/server-side.